README.md

```diff
@@ -108,7 +111,14 @@ Proc. IEEE, 102(9):1367-1388, 2014.
 - <a name="Vanarse16fnins"></a>Vanarse, A., Osseiran, A., Rassau, A,  
 *[A Review of Current Neuromorphic Approaches for Vision, Auditory, and Olfactory Sensors](http://dx.doi.org/10.3389/fnins.2016.00115)*,
 Front. Neurosci. (2016), 10:115.
-
+- <a name="Posch09Sensors"></a>
```

Comment:
Wrong formatting of the entry: the authors, title of the paper and publication venue should be in 3 different lines, like the rest of the entries.

Reply:
Also, I think the convention is to use lowercase for the suffix of the key value, "...09sensors"